dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -22,13 +22,16 @@ libraryRef=Shared Library
 libraryRef.desc=Identifies JCache provider files.
 
 uri=JCache configuration URI
-uri.desc=Vendor specific JCache configuration URI, which is passed to the JCache provider when obtaining the CacheManager.
+uri.desc=Vendor specific JCache configuration URI, which is passed to the JCache provider when obtaining the CacheManager. This setting is ignored when jCacheManagerRef is used.
 
 properties=JCache Configuration Properties
-properties.desc=List of vendor specific JCache configuration properties, which are passed to the JCache provider when obtaining the CacheManager.
+properties.desc=List of vendor specific JCache configuration properties, which are passed to the JCache provider when obtaining the CacheManager. This setting is ignored when jCacheManagerRef is used.
```

Comment:
List of vendor-specific JCache configuration properties, which are passed to the JCache provider when the CacheManager instance is obtained.

Reply:
Updated.